src/group_theory/quotient_group.lean

```diff
@@ -412,4 +412,57 @@ monoid_hom.to_mul_equiv
 
 end third_iso_thm
 
+
+section trivial
+
+/-These first few lemmas should maybe go somewhere else, where?-/
+
+variables {α : Type u} (r : α → α → Prop )
+
+lemma subsingleton_quot_iff_forall : subsingleton (quot r) ↔ ∀ a b, eqv_gen r a b :=
+begin
+  rw subsingleton_iff,
+  refine (surjective_quot_mk _).forall.trans (forall_congr $ λ a, _),
+  refine (surjective_quot_mk _).forall.trans (forall_congr $ λ b, _),
+  exact quot.eq,
+end
```

Comment:
I guess this probably belongs in `data/quot/basic`, but you'll end up waiting a very long time for a build once you put it there!

Reply:
Sorry, I cant find ``data/quot/basic.lean``. I can find ``data/quot.lean``, is that what you mean?

Reply:
Yes, that's what I meant!

Reply:
Ah yes so I tried this, and (not only was it slow) I was getting these errors:  "invalid tactic class 'tactic', 'tactic.save_info' has not been defined". Does it maybe not like that the proof has a begin ... end? (thats where the red lines show up)

Reply:
Can you push the change anyway and see what github has to say?

Reply:
I actually couldnt find the correct identifier for subsingleton_iff, I have this problem frequently, is there a quick way to find the correct identifier?